bb8/src/internals.rs

```diff
@@ -202,6 +231,7 @@ where
             conns: VecDeque::new(),
             num_conns: 0,
             pending_conns: 0,
+            inflight_gets: 0,
```

Comment:
Would prefer to rename this to `in_flight`.

Reply:
Can do!